config-windows.md

```diff
@@ -94,3 +94,23 @@ The following parameters can be specified:
         }
    }
 ```
+
+## <a name="configWindowsHyperV" />HyperV
+
+`hyperv` is an OPTIONAL field of the Windows configuration. If present, the container will be run with Hyper-V isolation. If omitted, the container will be run as a Windows Server container.
```

Comment:
@tianon Sure, updated. I have to be honest it does feel a little strong, but there's nothing closer.